tools/summaries/utils.py

```diff
@@ -58,6 +63,12 @@ def write_summaries_to_file(summaries: Dict, output_file: str):
   logging.info(f"Wrote summaries to {output_file}!")
```

Comment:
Remove `!`  (appreciate the enthusiasm though!)

Reply:
Removed `!`. Will be less enthusiastic in the future.